pkg/api/admin/openshiftcluster.go

```diff
@@ -139,6 +139,41 @@ const (
 	OutboundTypeLoadbalancer       OutboundType = "Loadbalancer"
 )
 
+// ResourceReference represents a reference to an Azure resource.
+type ResourceReference struct {
+	// The fully qualified Azure resource id.
+	ID string `json:"id,omitempty"`
+}
+
+// LoadbalancerProfile represents the profile of the cluster public loadbalancer.
+type LoadbalancerProfile struct {
+	// The desired managed outbound IPs for the cluster public load balancer.
+	ManagedOutboundIPs *ManagedOutboundIPs `json:"managedOutboundIps,omitempty"`
+	// The list of effective outbound IP addresses of the public load balancer.
+	EffectiveOutboundIPs []EffectiveOutboundIP `json:"effectiveOutboundIps,omitempty"`
```

Comment:
Will this be a list of resource IDs, or the IPs themselves? Here it sounds like they are IPs, but the comment on 162 suggests that they will be the resource IDs.

Reply:
If you follow the references all the way down to `ResourceReference` it clarifies that this is an object containing an azure ID.  The description is mostly verbatim from aks azure rest specs.
```
// ResourceReference represents a reference to an Azure resource.
type ResourceReference struct {
	// The fully qualified Azure resource id.
	ID string `json:"id,omitempty"`
```